scripts/20-install-dependencies-deb.part

```diff
@@ -27,7 +27,6 @@
   }; \
   { \
     # 20-install-dependencies-deb.part - Extract tools.
-    # ar on debian10 doesn't support '--output'
```

Comment:
The `ar` command on Ubuntu20 does support the `--output` option. Using that, we can otput directly to `aerospike/pkg` and remove the surrounding `pushd/popd`.